apisix/plugins/openid-connect.lua

```diff
@@ -54,6 +54,10 @@ local schema = {
             type = "string",
             default = "client_secret_basic"
         },
+        token_endpoint_auth_method = {
```

Comment:
Some CI tests are failing because of this schema update

![image](https://github.com/apache/apisix/assets/39619599/1adb1055-c229-4212-943e-e16c2a0fd7d1)

@monkeyDluffy6017 could you check this PR to see if the schema update makes sense and let me know if it's ok to update the expected test response? Thanks.